### DISCUSSION TITLE
Custom Fields Don't Show When Creating a New Asset

### DISCUSSION THREAD

**Discussion by Z06Luke** [snipe/snipe-it#10295] [General]
In Snipe-IT I have a few custom fields. The field in subject is CostPointID.

![image](https://user-images.githubusercontent.com/94076399/141181488-c9bd2ab3-ee03-4292-971b-38c62410054b.png)

When viewing or editing an asset the field is present, however when creating a new asset it does not show up. Is there a way to modify the field set when creating new assets?

![image](https://user-images.githubusercontent.com/94076399/141182187-22d6387e-1990-4791-ae97-df7c6d96a0d7.png)

![image](https://user-images.githubusercontent.com/94076399/141181932-c61334e3-af5f-447d-9a5b-c908c2a1a7c9.png)

**Comment by ShaunNeighbourSwale** [comment]
The custom fields should appear when you select the model associated with those custom fields.

**Reply by snipe** [reply]
Yep, exactly.
